charts/astarte-operator/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: astarte-operator
 description: A Helm chart for Kubernetes
 type: application
 
-version: 1.0.0-beta.1
-appVersion: 1.0.0-beta.1
+version: 1.0.0-beta.2
+appVersion: 1.0.0-beta.2
```

Comment:
Not sure about this. `beta-2` is not tagged yet. I'd rather suggest to update the `tag` [here](https://github.com/astarte-platform/astarte-kubernetes-operator/blob/release-1.0/charts/astarte-operator/values.yaml#L13) to `1.0-snapshot`. We may discuss how to handle the appVersion value.